navigation-breadcrumb/main.css

```diff
@@ -0,0 +1,37 @@
+:root {
+  --color-light-gray: #AEB0B5;
```

Comment:
Nitpicky, but maybe don't call out the color in the variable name?

Reply:
I opted to put those in as fallbacks for older browsers. Can remove or leave, no strong preference.

Reply:
I mis-read this previously. Yeah, we can remove the string `color`. Will update after I get back from lunch.

Reply:
I think color is ok, It was more about using light grey rather than something like secondary.